debian/control

```diff
@@ -168,8 +168,8 @@ Description: debugging symbols for ceph-base
 
 Package: cephadm
 Architecture: linux-any
-Depends: docker.io,
-         lvm2,
+Recommends: podman | docker.io
```

Comment:
seems podman is not yet packaged for debian.

so i was reading 

- the issue at https://github.com/containers/libpod/issues/1742 tracking the progress of packaging libpod for debian, and 
- the ITP bug at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=930440, 
- https://ftp-master.debian.org/new/libpod_1.6.4+dfsg-1.html

so, two observations

- podman packaging is close
- the package name is podman =)